include/xplat/Starboard/SmartTypes.h

```diff
@@ -18,6 +18,11 @@
 
 #include "Starboard/TypeTraits.h"
 
+// IwFree symbol must already be declared when not building on Windows
+#ifndef _WIN32
```

Comment:
You've already handled that in IWMalloc.h though.

---
In reply to: [72163245](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72163245) [](ancestors = 72163245,72162785)